thunder/executors/nvfuserex_impl.py

```diff
@@ -542,28 +544,10 @@ def __call__(self, *args):
         if self.store_inputs:
             self.last_inputs = args
 
-        if hasattr(fd, "multidevice_schedule"):
+        if IS_TORCH_DISTRIBUTED_AVAILABLE and any(isinstance(t, torch.distributed.tensor.DTensor) for t in args):
```

Comment:
```suggestion
        if dist.is_available() and any(isinstance(t, torch.distributed.tensor.DTensor) for t in args):
```
`dist.is_available()` is short and cheap already, and readers immediately know it's simply calling torch rather than some Thunder magic.